core/templates/dev/head/css/oppia.css

```diff
@@ -1328,7 +1328,7 @@ pre.oppia-pre-wrapped-text {
   margin-top: 60px;
   margin-left: auto;
   margin-right: auto;
-  max-width: 800px;
+  max-width: 844px;
   min-height: 500px;
```

Comment:
Done.

Note that max-width does come alphabetically before min-height, but I noticed that margin-top should go lower down, so I fixed that.